tests/unit/frameworks/continuous_learning/multihead/multihead_test.py

```diff
@@ -0,0 +1,170 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+Unit tests to ensure that target variables are being transformed properly according to
+the continual learning setup found in the benchmarks repository:
+https://github.com/GMvandeVen/continual-learning
+"""
+
+import unittest
+
+import torch
+
+from nupic.research.frameworks.continuous_learning.multihead.multihead import (
+    active_class_outputs,
+    get_active_classes,
+    get_target_transform,
+)
+from nupic.research.frameworks.pytorch.test_utils.fake_data import FakeDataLoader
+from nupic.torch.modules import Flatten
+
+
+def setup(scenario):
```

Comment:
`scenario` arg is not being used, it could be renamed to `setup_dataloaders`

Reply:
Done.